src/rust/receiver-mock/src/router.rs

```diff
@@ -153,26 +152,25 @@ receiver_mock_logs_bytes_count {}
                 match content_type {
                     // Metrics in carbon2 format
                     "application/vnd.sumologic.carbon2" => {
-                        metrics::handle_carbon2(lines, address, &stats);
+                        metrics::handle_carbon2(lines, address, &stats, opts.print_opts);
                     }
                     // Metrics in graphite format
                     "application/vnd.sumologic.graphite" => {
-                        metrics::handle_graphite(lines, address, &stats);
+                        metrics::handle_graphite(lines, address, &stats, opts.print_opts);
                     }
                     // Metrics in prometheus format
                     "application/vnd.sumologic.prometheus" => {
-                        metrics::handle_prometheus(lines, address, &stats);
+                        metrics::handle_prometheus(lines, address, &stats, opts.print_opts);
```

Comment:
why not to pass all opts?

Reply:
Only pass what's needed - I try to stick to this rule. If we'd have a struct with 100 fields why should `metrics::handle_prometheus` know about them? Its only concern is whether to print "stuff"

Reply:
Yeah, but if for some reason we will want to pass some extra argument to the prometheus metrics, the function signatures will start to differ between therm and it's going to be messy, but maybe I'm trying to over-engineer this 🤔

Reply:
That's a valid point but! it feels like this very interesting, handy function that's not used by anyone but it's checked in in the repo "just in case".

If we are going to need to change the signature of those then we'll think about this :)